pkg/cmd/gist/view/view.go

```diff
@@ -107,6 +102,11 @@ func viewRun(opts *ViewOptions) error {
 
 	showFilenames := len(gist.Files) > 1
 
+	cs := opts.IO.ColorScheme()
+	if gist.Description != "" && showFilenames {
```

Comment:
The new condition is going to make the rendering of gist description toggle on/off based on the number of files in a gist. This behavior might be confusing to users.

How about, we do not render the description when a filename is specified?

Reply:
change condition to check opts.Filename